### PR TITLE
Update mainnet and testnet urls in index page in service docs v0 6

### LINKED ISSUE #1627: Update http mainnet and testnet url in doc page
### Description
A documentation page in lisk-service is showing websocket endpoint in place of http endpoints

```
Current endpoints:
    main-net: wss://service.lisk.com/api/
    test-net: wss://testnet-service.lisk.com/api/

Expected endpoints:
    main-net: https://service.lisk.com/api/v2/.
    test-net:  https://testnet-service.lisk.com/api/v2/.
```

### Affected branch / URL
https://lisk.com/documentation/lisk-service/index.html#the-http-api

### Additional resources

### PR DESCRIPTION
**Pull request by chris529**
Closes #1627

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -7,10 +7,10 @@ Mona Bärenfänger <mona@lightcurve.io>
 :imagesdir: ../assets/images
 :page-no-previous: true
 
-:url_api_mainnet: wss://service.lisk.com/api/
-:url_api_testnet: wss://testnet-service.lisk.com/api/
-:url_rpc_api_mainnet: wss://service.lisk.com/rpc-v2
-:url_rpc_api_testnet: wss://testnet-service.lisk.com/rpc-v2
+:url_api_mainnet: https://service.lisk.com/api/v3/
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:url_api_mainnet: https://service.lisk.com/api/v2/
```

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -7,10 +7,10 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+:url_api_testnet: https://testnet-service.lisk.com/api/v3/
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:url_api_testnet: https://testnet-service.lisk.com/api/v2/
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -7,10 +7,10 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+:url_rpc_api_mainnet: wss://service.lisk.com/rpc-v3
+:url_rpc_api_testnet: wss://testnet-service.lisk.com/rpc-v3
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:url_rpc_api_mainnet: wss://service.lisk.com/rpc-v2
:url_rpc_api_testnet: wss://testnet-service.lisk.com/rpc-v2
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -105,10 +105,10 @@ Lisk Testnet::
 
 ==== Example: Request data with curl
 
-.Example request: Obtain a list of all 101 currently forging delegates
+.Example request: Obtain a list of generators in the current round
 [source,bash]
 ----
-curl -X GET "http://localhost:9901/api/v2/forgers" -H  "accept: application/json"
+curl -X GET "http://localhost:9901/api/v3/generators" -H  "accept: application/json"
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
curl -X GET "http://localhost:9901/api/v2/forgers" -H  "accept: application/json"
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -186,9 +186,9 @@ const jsome = require('jsome'); // Prettifies the JSON output
 jsome.params.colored = true;
 
 // Use local Service node
-const WS_RPC_ENDPOINT = 'ws://localhost:9901/rpc-v2';
+const WS_RPC_ENDPOINT = 'ws://localhost:9901/rpc-v3';
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
const WS_RPC_ENDPOINT = 'ws://localhost:9901/rpc-v2';
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -186,9 +186,9 @@ const jsome = require('jsome'); // Prettifies the JSON output
...
 //Use public Service node
-//const WS_RPC_ENDPOINT = "wss://service.lisk.com/rpc-v2";
+//const WS_RPC_ENDPOINT = "wss://service.lisk.com/rpc-v3";
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
//const WS_RPC_ENDPOINT = "wss://service.lisk.com/rpc-v2";
```

File: modules/ROOT/pages/reference/api-testnet.adoc [hunk 7]
```diff
@@ -3,4 +3,4 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: Interactive HTTP API reference of Lisk Service (Testnet).
 :page-no-next: true
 :page-layout: swagger
-:page-swagger-url: https://testnet-service.lisk.com/api/v2/spec
+:page-swagger-url: https://testnet-service.lisk.com/api/v3/spec
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/reference/api-testnet.adoc]:
```suggestion
:page-swagger-url: https://testnet-service.lisk.com/api/v2/spec
```

File: modules/ROOT/pages/reference/api.adoc [hunk 8]
```diff
@@ -3,4 +3,4 @@ Mona Bärenfänger <mona@lightcurve.io>
 :description: Interactive HTTP API reference of Lisk Service (Mainnet).
 :page-no-next: true
 :page-layout: swagger
-:page-swagger-url: https://service.lisk.com/api/v2/spec
+:page-swagger-url: https://service.lisk.com/api/v3/spec
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/reference/api.adoc]:
```suggestion
:page-swagger-url: https://service.lisk.com/api/v2/spec
```

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -105,10 +105,10 @@ Lisk Testnet::
 
 ==== Example: Request data with curl
 
-.Example request: Obtain a list of all 101 currently forging delegates
+.Example request: Obtain a list of generators in the current round
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/index.adoc]:
```suggestion
.Example request: Obtain a list of all 101 currently forging delegates
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 🎉

**Pull request merged**